components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/exception/MethodNotAllowedException.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Fixed in #1494(https://github.com/wso2/carbon-apimgt/commit/c3facbf86bfb7c4cd023d0d0b423829c52ee290c)